### ISSUE TITLE
Change entry point of shader when compiling shaders

### ISSUE THREAD

**Issue by jaebaek** [google/amber#189]
Based on [description from glslang](https://github.com/google/shaderc/blob/master/libshaderc/include/shaderc/shaderc.hpp#L367), 
glslang considers `main` as entry point.

`entrypoint` command requires the change of entry point name when compiling shaders.
If we do not change, SPIRV based script would be fine but GLSL based script does not work.

**Comment by dj2** [comment]
I don't think this is necessary. The shader is the shader, if you call entrypoint on something that isn't in the shader that should be an error. We don't want to start trying to re-write the shaders based on the test commands.
